packages/js-moi-manifest/src.ts/manifest.ts

```diff
@@ -279,4 +280,182 @@ export class ManifestCoder {
+                        ...element.data,
+                    ]);
+                    schema = Schema.PISA_CONSTANT_SCHEMA;
+                    break;
+                }
+
+                case "typedef": {
+                    buff = new Uint8Array([
+                        WireType.WIRE_WORD,
+                        ...element.data,
+                    ]);
+                    schema = Schema.PISA_TYPEDEF_SCHEMA;
+                    break;
+                }
+
+                case "state": {
+                    buff = new Uint8Array([
+                        WireType.WIRE_PACK,
+                        ...element.data,
+                    ]);
+                    schema = Schema.PISA_STATE_SCHEMA;
+                    break;
+                }
+
+                case "routine": {
+                    buff = new Uint8Array([
+                        WireType.WIRE_PACK,
+                        ...element.data,
+                    ]);
+                    schema = Schema.PISA_ROUTINE_SCHEMA;
+                    break;
+                }
+
+                case "method": {
+                    buff = new Uint8Array([
+                        WireType.WIRE_PACK,
+                        ...element.data,
+                    ]);
+                    schema = Schema.PISA_METHOD_SCHEMA;
+                    break;
+                }
+
+                case "class": {
+                    buff = new Uint8Array([
+                        WireType.WIRE_PACK,
+                        ...element.data,
+                    ]);
+                    schema = Schema.PISA_CLASS_SCHEMA;
+                    break;
+                }
+
+                case "event": {
+                    buff = new Uint8Array([
+                        WireType.WIRE_PACK,
+                        ...element.data,
+                    ]);
+                    schema = Schema.PISA_EVENT_SCHEMA;
+                    break;
+                }
+
+                default: 
+                    ErrorUtils.throwError(`Unsupported kind: ${element.kind}`, ErrorCode.UNSUPPORTED_OPERATION);
+            }
+
+            if (schema == null || buff == null) {
+                ErrorUtils.throwError("Invalid schema or buffer", ErrorCode.UNEXPECTED_ARGUMENT);
+            }
+
+            const polo = new Depolorizer(buff);
+            element.data = polo.depolorize(schema);
+
+            if (["routine", "method"].includes(element.kind)) {
+                for (const [key, value] of Object.entries(
+                    element.data.executes
+                )) {
+                    // If value of `Instruction[key]` is empty, remove it
+                    if (value === "") {
+                        delete element.data.executes[key];
+                    }
+
+                    // If value of `Instruction` is empty, remove it
+                    if (Array.isArray(value) && value.length === 0) {
+                        delete element.data.executes[key];
+                    }
+
```

Comment:
Why do we have to delete it.

Reply:
<img width="295" alt="image" src="https://github.com/user-attachments/assets/255b0081-bdf3-4897-ba75-4e434f5e1a89">

As executes can have fields `bin`, `asm`. These are OPCODE array thus they are not present and length is zero. I'm removing it from manifest json